tsconfig.json

```diff
@@ -1,20 +1,15 @@
 {
   "compilerOptions": {
-    "target": "es2021",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "ESNext",
```

Comment:
I wonder if it's better to target a specific version rather than ESNext here. Maybe ES2023 if we're updating to node 22?
https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping

Reply:
Also interestingly from your link 

> After TypeScript 5.9 is released, it is recommended to set it to node20 instead of `ESXXXX`